### ISSUE TITLE
add a way to load and manipulate an off-screen web page

### ISSUE THREAD

**Issue by insinfo** [jnschulze/flutter-webview-windows#84]
I need a way to load this web page https://spe.riodasostras.rj.gov.br/iptu/guia.aspx off the screen and fill and send the form and get the result page and render inside the Scaffold.

Is there any way to do this with flutter-webview-windows ?

![image](https://user-images.githubusercontent.com/12227024/150037546-e06fd66f-465b-49be-94dd-22cfba2a25f5.png)

![image](https://user-images.githubusercontent.com/12227024/150037589-685844fe-2cd3-4b1b-aa4d-b2014833a679.png)

**Comment by jnschulze** [comment]
This should be possible by using `ExecuteScript` and the message-passing related methods of `WebviewController`.
However, from an architecture point of view, I’d delegate such functionality to some backend app.

**Comment by insinfo** [comment]
I would like to do this in the backend, but I haven't found a lib to work with off-screen webview with dart, does this lib work with dart-vm? since my app backend is written in dart, I would like to access the DOM and execute events as in the DotNetBrowser lib https://dotnetbrowser-support.teamdev.com/docs/guides/gs/dom.html#dom.

How does "message-passing" work? Does this allow passing mouse and keyboard events to the webview to eg click a button and or fill in an input ?

**Comment by jnschulze** [comment]
> How does "message-passing" work? Does this allow passing mouse and keyboard events to the webview to eg click a button and or fill in an input ?

No, it’s just about passing JSON-objects from Dart to JS and vice versa.
